packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -273,8 +273,25 @@ function isElementInAnyScope(element: Element, scopes: Set<RefObject<HTMLElement
   return false;
 }
 
+const focusScopeDataAttrNames = [
+  'data-focus-scope-start',
+  'data-focus-scope-end'
+];
+
+function isFocusScopeDirectChild(scopes: HTMLElement[]) {
+  return scopes.some(scope => focusScopeDataAttrNames.some(name => scope.getAttribute(name) !== null));
+}
+
+function isFocusScopeNestedChild(element: Element) {
+  return focusScopeDataAttrNames.some(name => element.querySelector(`[${name}]`));
```

Comment:
lets have the two signatures match
```suggestion
function isFocusScopeNestedChild(scopes: Element[]) {
  return scopes.some(scope => focusScopeDataAttrNames.some(name => element.querySelector(`[${name}]`)));
```

Reply:
or the other way around, then line 290 could be like this instead
```
return scopes.some(scope => isFocusScopeDirectChild(scope) || isFocusScopeNestedChild(scope));
```

Reply:
https://github.com/adobe/react-spectrum/pull/2139/commits/f3e9704b804c306a2d8a293c4348349f4763369e